aiidalab_widgets_base/wizard.py

```diff
@@ -0,0 +1,183 @@
+"""The wizard application allows the implication of a Wizard-like GUI.
+
+Authors:
+
+    * Carl Simon Adorf <simon.adorf@epfl.ch>
+"""
+from enum import Enum
+
+import traitlets
+import ipywidgets as ipw
+
+
+class WizardApp(ipw.VBox):
+    class State(Enum):
+        "Every step within the WizardApp must have this traitlet."
+        INIT = 0  # implicit default value
+
+        CONFIGURED = 1
+        READY = 2
+        ACTIVE = 3
+        SUCCESS = 4
+
+        # All error states have negative codes
+        FAIL = -1
+
+    #     The state of a step can no longer be changed once
+    #     it is one of the following sealed states.
+    SEALED_STATES = [State.SUCCESS, State.FAIL]
+
+    ICON_SEPARATOR = "\u2000"  # en-dash  (placed between title and icon)
+
+    ICONS = {
+        State.INIT: "\u25cb",
+        State.CONFIGURED: "\u25ce",
+        State.READY: "\u25ce",
```

Comment:
```suggestion
        State.READY: "\u25ce",
        State.CONFIGURED: "\u25cf",
```

What I understood from your example is that the `CONFIGURED` state is more "complete".

I prefer the following symbols:

`INIT` - empty circle, ready to be filled in. [as it is now]
`READY` - two concentric circles, in the process of filling [as it is now]
`CONFIGURED` - full circle, everything is provided [taken from the `SUCCESS`]
`ACTIVE` - rotating circle [ as it is now]
`SUCCESS` - check mark
`FAILURE` - cross

Reply:
Done.